at-tanvo/20180719064743_change_gender.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeGender < ActiveRecord::Migration[5.2]
+  def up
+    User.where(gender: 'nam').update_all(gender: '0')
+    User.where(gender: 'nu').update_all(gender: '1')
+    User.where(gender: 'khac').update_all(gender: '2')
+    change_column :users, :gender, :tinyint
+  end
+
+  def down
+    change_column :users, :gender, :string
+    User.where(gender: '0').update_all(gender: 'nam')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/766d088c93f3ec3982f71e8b38d34ce0e8fb5420/at-tanvo/20180719064743_change_gender.rb#L11"></span>Rails/SkipsModelValidations: Avoid using <code>update_all</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>